language/src/modules/Language.rb

```diff
@@ -363,70 +362,129 @@ def GetLanguageExtensionFilename(language)
       filename
     end
 
+    # Downloads inst-sys extension for a given language
+    # including giving a UI feedback to the user
+    #
+    # @param [String] language, e.g. 'de_DE'
+    def integrate_inst_sys_extension(language)
+      log.info "integrating translation extension..."
+
+      # busy message
+      Popup.ShowFeedback(
+        "",
+        _("Downloading installation system language extension...")
```

Comment:
NP: Sounds too long to me, might not fit or look nice in 80x25 terminal... Think about some extra long translations! Moreover the popup can close before you finish reading.

Simple `Downloading language extension...` is enough IMHO.